examples/standalone/runprgm/src/main.c

```diff
@@ -0,0 +1,41 @@
+#include <tice.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+int prgm2_ret(void *data, int retval)
+{
+    printf("executed prgm 2.\n");
+    printf("retval: %d", retval);
+    (void)data;
```

Comment:
Would it be better to use `__attribute__((unused))` for this? Or are we avoiding C extensions in the example code?

Reply:
Eh, we see this very often, looks fine to me.